src/routes/landing.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+
+class Landing extends React.Component {
+  render() {
+    return (
+      <View style={styles.landing}>
+        <View style={styles.vContainer}>
```

Comment:
If you find yourself copy/pasting the same thing over and over again, it means it wants to be a component. How about `VContainer` and `HContainer` components? (not set on that naming)

Reply:
Good call. I will revisit this.